.cake/Publish-MsBuild-Folder.cake

```diff
@@ -14,7 +16,7 @@ Task("Publish:MsBuild")
         MSBuild(webProject.ProjectFilePath, c => c
             .SetConfiguration(config.Solution.BuildConfiguration)
             .SetVerbosity(Verbosity.Quiet)
-            .UseToolVersion(MSBuildToolVersion.VS2017)
+            .UseToolVersion(config.MsBuildToolVersion)
```

Comment:
Is `MsBUildToolVersion` a property added into the base `Configuration` class? - this is introducing a property into configuration that is only used by specific tasks.  The `TaskParameters` dictionary should be used to set and get task specific parameters, where the task itself is aware of a task parameter with a given key.  See the Npm-RunScript or CopyArtifacts as examples.

Reply:
The `MSBuildToolVersion` is a property on a partial `Configuration` class defined in the Build-MsBuild task.

As discussed I'm going to change this to use the `TaskParameters` dictionary instead.

Reply:
Fixed with https://github.com/stormid/cake-mix/pull/18/commits/bd5a5791cf7fa1b475dfa76eeb6a62bc131e89c6